lib/OpenQA/Constants.pm

```diff
@@ -20,11 +20,15 @@ use constant MAX_TIMER => 100;
 # Time verification to be use with WORKERS_CHECKER_THRESHOLD.
 use constant MIN_TIMER => 20;
 
+# The smallest distinguishable time difference of timestamps in the database in seconds
+use constant DB_TIMESTAMP_ACCURACY => 1;
```

Comment:
where does this number come from? Is it something we can configure? Not knowing yet where this is used having a granularity of just one second seems to be a bit vague for some use cases. Maybe we should/can change?

Reply:
We are actually using the PostgreSQL data type `timestamp` which has an accuracy of one microsecond. However, it seems that this is not really utilized by our code and DBIx which only deal with seconds. Just look at any table with timestamps. There are no values which are not whole seconds for columns like `t_updated`.

I thought it would be better to define a constant here instead using a magic number. Maybe the comment should be improved, e.g. "The smallest time difference of database timestamps we usually distinguish in seconds"?